js/sagas/index.js

```diff
@@ -5,6 +5,7 @@ import {
     togglesSagas,
     sykeforlopsPerioderSagas,
 } from 'digisyfo-npm';
+import { motebehovSagasAT as motebehovSagas } from 'moter-npm';
```

Comment:
Hvis det er en egen saga for arbeidstakere, kan vel den like gjerne flyttes inn her og fjernes fra moter-npm? Slik du snakket om i går?